### ISSUE TITLE
R functions to write tests on

### ISSUE THREAD

**Issue by lilyclements** [IDEMSInternational/cdms.products#86]
Check if we can remove the following R files without it impacting elsewhere:

- [ ] convert_to_dec_deg.R 
- [ ] get_lon_from_data.R
- [ ] get_lat_from_data.R
- [ ] lat_lon_dataframe.R

Need to create tests on:
- [ ] Export functions (possibly: call prepare function, export as CSV. Run export function (creates second CSV). Read in both CSVs, and check same data frame)
- [ ] Plot functions
- [ ] `make_factor`
- [ ] `prepare_geoclim_*` functions
- [ ] `spells` function
- [ ] CPT (perhaps split this into a prepare and export function)
- [ ] `summarise_inventory_data` function
- [ ] `wwr_export` (should output textfile should be this text. so can write it and compare)

**Comment by lilyclements** [comment]
@lloyddewit how are you testing export functions in python?

**Comment by lloyddewit** [comment]
> @lloyddewit how are you testing export functions in python?

@lilyclements I duplicate the tests in the `test_*.R` files [here](https://github.com/IDEMSInternational/cdms.products/tree/main/tests/testthat).

Currently, in the main branch, there aren't any tests for the `export_*.R` functions.

However, @HawardKetoyoMsatsi has submitted several PRs that propose tests for the export functions. In the Python layer, I initially implemented the tests in PR #78 . I'll also implement the tests in the other PRs, plus any other tests that are added.

In summary, if you (or @HawardKetoyoMsatsi ) can add new export tests, then I'll add these tests to the Python layer.

**Comment by lloyddewit** [comment]
@lilyclements I added a new task to the PR description to add a test for:

> - [ ] `export_climat_messages` function
